### PR TITLE
[feature] setup formatters for code auditors

### PR DESCRIPTION
**Pull request by za4emnik**
https://garage.atlassian.net/browse/PEO-90

### CODE REVIEW

**Comment by za4emnik** [comment]
Doc: https://docs.google.com/spreadsheets/d/1RmlEnx6XSCgPeN6YeHuFEGyOumwFQJLAomz8RJNaHhI/edit?usp=sharing

File: lib/inquisition/auditors/backend/rails_best_practices/formatter.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+module Inquisition
+  module Auditors
+    module Backend
+      module RailsBestPractices
+        class Formatter < Core::BaseFormatter
+          REPORT_KEYS = {
+            linterable_name: 'filename',
```
Comment (Eversilence) [inline lib/inquisition/auditors/backend/rails_best_practices/formatter.rb]:
```suggestion
            lintable_name: 'filename',
```

linterable is incorrect wording, there is no such word in english language :)

File: lib/inquisition/auditors/backend/rails_best_practices/installer.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+module Inquisition
+  module Auditors
+    module Backend
+      module RailsBestPractices
+        class Installer < Auditors::Installer
+          class << self
+            private
+
+            def configers
```
Comment (Eversilence) [inline lib/inquisition/auditors/backend/rails_best_practices/installer.rb]:
```suggestion
            def configurers
```
again, no such word exists

File: lib/inquisition/auditors/backend/ruby_lint/formatter.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+module Inquisition
+  module Auditors
+    module Backend
+      module RubyLint
+        class Formatter < Core::BaseFormatter
+          REPORT_KEYS = {
+            linterable_name: 'file',
```
Comment (Eversilence) [inline lib/inquisition/auditors/backend/ruby_lint/formatter.rb]:
```suggestion
            lintable: 'file',
```

File: lib/inquisition/auditors/backend/ruby_lint/formatter.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+          private
+
+          def build_errors
+            @build_errors ||= raw_errors.map(&method(:build_error))
+          end
+
+          def build_error(error)
+            {
+              linterable_name: error[REPORT_KEYS[:linterable_name]],
```
Comment (Eversilence) [inline lib/inquisition/auditors/backend/ruby_lint/formatter.rb]:
```suggestion
              lintable_name: error[REPORT_KEYS[:linterable_name]],
```

File: lib/inquisition/auditors/backend/ruby_lint/formatter.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+              linterable_type: linterable_type_default,
```
Comment (Eversilence) [inline lib/inquisition/auditors/backend/ruby_lint/formatter.rb]:
```suggestion
              lintable_type: linterable_type_default,
```

File: lib/inquisition/auditors/frontend/i18n_tasks/formatter.rb [hunk 6]
```diff
@@ -0,0 +1,39 @@
+module Inquisition
+  module Auditors
+    module Frontend
+      module I18nTasks
+        class Formatter < Core::BaseFormatter
+          REPORT_KEYS = {
+            linterable_name: 'filename',
```
Comment (Eversilence) [inline lib/inquisition/auditors/frontend/i18n_tasks/formatter.rb]:
```suggestion
            lintable_name: 'filename',
```

**Review by dzemlianoi-double — APPROVED** [review]

**Pull request merged**